packages/@okta/vuepress-site/docs/guides/migrate-to-okta/prerequisites/index.md

```diff
@@ -0,0 +1,157 @@
+
+For some helpful information on this, see:
+
+* [How to Keep PII Secure while Migrating Users to the Cloud](https://www.okta.com/gdpr/) (Okta security blog post)
+* [Prepare your organization for the GDPR](https://www.okta.com/gdpr/) (Okta data security page)
+
+## Get ready to create a plan - Gather information
+
+Before being able to create a plan, you must gather information about your environment and decide on specific goals of the migration project. This includes making a list of the types of users you want to migrate and the applications and other resources that they will need to access through Okta.
+
+### Identify the users to migrate
+
+Start by identifying the types of users you have, based on the broad kinds of access they will need. For example:
+
+* Full and part time employees of your company
+* Customers
+* Partners
+* Contractors
+
+How many of each type of user do you have? Understanding this will help estimate the amount of time you’ll need when implementing your migration.
+
+Also identify any sub-groups currently in use for all of your user types. For example, are your employees grouped into sales, marketing, and accounting groups? What about your customers?
+
+### Locate your existing user profile data
+
+Your user data could be in Active Directory, LDAP, or any number of on-premise or cloud-based applications, but most importantly you need to know exactly where the data is and how to access it. This also requires that you have permission to access and use the data. You’ll also need to know which system is the source of truth for each data element. For example, various systems might maintain a primary email address for each of your users.
+
+Here are the key points to remember:
+
+* Record each system that currently holds user profile data.
+* Find out who is responsible for access to each of those systems.
+* Obtain permission and access to extract the necessary data.
+* Decide if user credentials, groups, and group memberships will be migrated.
+* Identify which user attributes are personally identifiable information (PII).
+* Define all user attributes that will be migrated to Okta, and the source of truth for each one.
+
+**Note:** The source of truth for attributes at this stage of the migration process means only that we want to clearly define where the user data is coming from for the migration itself, not where it will be managed after the migration. After migration, Okta is the default [profile master](https://help.okta.com/en/prod/Content/Topics/Directory/eu-profile-masters.htm), but this can be modified.
```

Comment:
```suggestion
> **Note:** The source of truth for attributes at this stage of the migration process means only that we want to clearly define where the user data is coming from for the migration itself, not where it will be managed after the migration. After migration, Okta is the default [profile master](https://help.okta.com/en/prod/okta_help_CSH.htm#csh-profile-masters), but this can be modified.
```